toolkits/landcover/impl/landsat8.js

```diff
@@ -41,16 +42,19 @@ var QA_BAND = 'pixel_qa';
  * Returns a new Landsat 8 SR dataset instance.
  *
  * @constructor
+ * @param {ee.ImageCollection=} collection The Landsat image collection backing
+ *    this dataset. If not specified, Landsat 8 SR will be used.
  * @return {!Landsat8}
  */
-var Landsat8 = function() {
+var Landsat8 = function(collection) {
```

Comment:
If we treat datasets as builders this arg can go away along with the clone() operations. Wdyt?